source/common/res/features/budget-daily-spending/main.js

```diff
@@ -0,0 +1,127 @@
+(function poll() {
+  if (typeof ynabToolKit !== 'undefined' && ynabToolKit.pageReady === true) {
+    ynabToolKit.dailySpending = (function () {
+      return {
+        budgetView: ynab.YNABSharedLib.getBudgetViewModel_AllBudgetMonthsViewModel()._result,
```

Comment:
So I'm trying to move away from using this because even though I've never seen `_result` not set, that doesn't mean it'll always be set since it's only set when the promise finishes.

I recently found this object: `ynab.YNABSharedLib.defaultInstance.entityManager`

Could you try using that to access the masterCategoriesCollection?